src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -35,7 +35,8 @@ public class RoatpApplicationPreambleController : Controller
         private readonly IUsersApiClient _usersApiClient;
         
         private const string ApplicationDetailsKey = "Roatp_Application_Details";
-        
+        private const string GetHelpSubmittedForPageKey = "Roatp_GetHelpSubmitted_{0}";
```

Comment:
This looks more like a format string than a key

Reply:
Renamed constant name